examples/Planck/Planck.ino

```diff
@@ -0,0 +1,181 @@
+),
+
+/* Plover layer (http://opensteno.org)
+ * ,-----------------------------------------------------------------------------------.
+ * |   #  |   #  |   #  |   #  |   #  |   #  |   #  |   #  |   #  |   #  |   #  |   #  |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |   S  |   T  |   P  |   H  |   *  |   *  |   F  |   P  |   L  |   T  |   D  |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |   S  |   K  |   W  |   R  |   *  |   *  |   R  |   B  |   G  |   S  |   Z  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Exit |      |      |   A  |   O  |             |   E  |   U  |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+
+[_PLOVER] = KEYMAP(
+    LSHIFT(Key_1),    LSHIFT(Key_1),    LSHIFT(Key_1),    LSHIFT(Key_1),    LSHIFT(Key_1),    LSHIFT(Key_1),    LSHIFT(Key_1),    LSHIFT(Key_1),    LSHIFT(Key_1),    LSHIFT(Key_1),    LSHIFT(Key_1),    LSHIFT(Key_1)   ,
+    XXX, Key_Q,    Key_W,    Key_E,    Key_R,    Key_T,    Key_Y,    Key_U,    Key_I,    Key_O,    Key_P,    Key_LeftBracket,
+    XXX, Key_A,    Key_S,    Key_D,    Key_F,    Key_G,    Key_H,    Key_J,    Key_K,    Key_L,    Key_Semicolon, Key_Quote,
+    LockLayer(_QWERTY), XXX, XXX, Key_C,    Key_V,    XXX, XXX, Key_N,    Key_M,    XXX, XXX, XXX
+),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |      | Reset|      |      |      |      |      |      |      |      |      |  Del |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      |      |Aud on|Audoff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|Plover|      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |Voice-|Voice+|Mus on|Musoff|MIDIon|MIDIof|      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+//[_ADJUST] = KEYMAP(
+//    ___, RESET,   DEBUG,    RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, Key_Delete ,
+//    ___, ___, MU_MOD,  AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, LockLayer(_QWERTY),  LockLayer(_COLEMAK), LockLayer(_DVORAK),  LockLayer(_PLOVER),  ___,
+//    ___, MUV_DE,  MUV_IN,  MU_ON,   MU_OFF,  MI_ON,   MI_OFF,  TERM_ON, TERM_OFF, ___, ___, ___,
+//    ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___
+//)
+);
+/* *INDENT-ON* */
+
+
+
+KALEIDOSCOPE_INIT_PLUGINS(Macros);
+
+void setup() {
+  Kaleidoscope.setup();
+  Serial.begin(9600);
+}
+
+void loop() {
+  Kaleidoscope.loop();
+}
+
```

Comment:
Nitpicking, but there are too many empty lines here. Can they be collapsed into only a single one?